stream-chat-android-offline/api/stream-chat-android-offline.api

```diff
@@ -80,13 +80,13 @@ public abstract interface class io/getstream/chat/android/livedata/ChatDomain {
 }
 
 public final class io/getstream/chat/android/livedata/ChatDomain$Builder {
+	public fun <init> (Landroid/content/Context;Lio/getstream/chat/android/client/ChatClient;)V
 	public fun <init> (Landroid/content/Context;Lio/getstream/chat/android/client/ChatClient;Lio/getstream/chat/android/client/models/User;)V
-	public synthetic fun <init> (Landroid/content/Context;Lio/getstream/chat/android/client/ChatClient;Lio/getstream/chat/android/client/models/User;ILkotlin/jvm/internal/DefaultConstructorMarker;)V
-	public fun <init> (Lio/getstream/chat/android/client/ChatClient;Landroid/content/Context;)V
```

Comment:
It looks like we lost this constructor that had two parameters, `ChatClient` and then `Context`. This will break any calls in client code that look like `ChatDomain.Builder(client, applicationContext)`, just like they broke our docs samples. We should keep this constructor, and make it deprecated if we really want to make everyone move over to using `ChatDomain.Builder(applicationContext, client)` instead.